components/organisms/UserSettingsPage/user-settings-page.tsx

```diff
@@ -433,19 +434,22 @@ const UserSettingsPage = ({ user }: UserSettingsPageProps) => {
             >
               Update Preferences
             </Button>
-          </div>
+          </fieldset>
           {userInfo && (
             <>
               <div>
                 {!coupon ? (
                   <div className="flex flex-col order-first gap-6 md:order-last">
                     <div className="flex flex-col gap-3">
-                      <label className="text-2xl font-normal">Developer Pack</label>
+                      <label htmlFor="verify-account-button" className="text-2xl font-normal">
+                        Developer Pack
+                      </label>
```

Comment:
This shouldn't be a label. The button in the DeveloperPackForm component mentioned below has button text, so no need to link it to this.

```suggestion
                      <h2 className="text-2xl font-normal">
                        Developer Pack
                      </h2>
```

Reply:
I have not pushed it yet however I have made these changes as you suggested previously https://github.com/open-sauced/app/pull/4152#discussion_r1792026462 but I was waiting for your response on this https://github.com/open-sauced/app/pull/4152#discussion_r1799670315

Reply:
I'll take a peek at that comment, but geel free to push the other change requests.

Reply:
I have pushed the change requests and the remaining fixes will be done after your response to above mentioned comment